fr.free.Qccrypt.appdata.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2023 Philippe Beaureilles -->
+<component type="desktop-application">
+  <id>fr.free.Qccrypt</id>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>GPL-2.0+</project_license>
+  <name>Qccrypt</name>
+  <summary>QT based graphical frontend for ccrypt tool</summary>
+  <developer_name>Philippe Beaureilles</developer_name>
+  <description>
+    <p>QT based graphical frontend for ccrypt tool.</p>
```

Comment:
this should also explain what ccrypt is.

Reply:
Ok, I will add a description for ccrypt.